R/R/executeInSQL.R

```diff
@@ -202,40 +243,49 @@ execute <- function(connectionString, script)
         # 3. The warnings of the function
         # 4. The errors of the function
         # We raise warnings and errors, print any output, and return the actual function results to the user
-
-        if (len > 1) {
+        #
+        if (len > 1)
+        {
             output <- resVal[[2]]
-            for(o in output) {
+            for(o in output)
+            {
                 cat(paste0(o,"\n"))
             }
         }
-        if (len > 2) {
+
+        if (len > 2)
+        {
             warnings <- resVal[[3]]
-            for(w in warnings) {
+            for(w in warnings)
+            {
                 warning(w)
             }
         }
-        if (len > 3) {
+
+        if (len > 3)
+        {
             errors <- resVal[[4]]
-            for(e in errors) {
+            for(e in errors)
+            {
                 stop(paste0("Error in script: ", e))
             }
         }
-        return(resVal)
-    } else {
-        return(res)
+
+        res <- resVal
     }
+
+    return(res)

```

Comment:
what happens if res is not initialized ? what do we return ?

Reply:
If res is not initialized, then there will be an error in the tryCatch above, and the function will error out: 

        queryResult <- dbSendQuery(hodbc, script, ...)
        res <- dbFetch(queryResult)	        
        binVal <- res$returnVal